.gitignore

```diff
@@ -0,0 +1 @@
+nohup.out
```

Comment:
Where did this come from?

Reply:
I made the change. It came from me wanting to use something that can do multiple categories (treasurer requires reregistering to do that). I plan to post some loot_ registration mods as needed (as separate repos). Then structures or mobs can have there own per-type groups (such as pyramid, mummy, or broader such as undead).

Reply:
Ahh, ok, that comes from nohup, such as `nohup geany init.lua` -- if its too much of an edge case, feel free to remove it.

Reply:
Could you remove it from the PR? I'll merge it afterwards.

Reply:
Done.